snuba/datasets/profiles_processor.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+from typing import Any, Mapping, Optional
+
+from snuba.consumers.types import KafkaMessageMetadata
+from snuba.processor import InsertBatch, MessageProcessor, ProcessedMessage
+
+
+class ProfilesProcessor(MessageProcessor):
+    def process_message(
+        self, message: Mapping[str, Any], metadata: KafkaMessageMetadata
+    ) -> Optional[ProcessedMessage]:
+        try:
+            processed = {
+                "organization_id": message["organization_id"],
+                "project_id": message["project_id"],
+                "transaction_id": message["transaction_id"],
```

Comment:
IIRC the reason we do have this problem in other processors is because Sentry always deals with transaction_ids and other uuids as hexadecimal strings with length of 32 (i.e. without the dashes). So we typically would do something like `str(uuid.UUID(transaction_id))` to add the dashes before inserting to ClickHouse.